test/integration/deprecations-test.js

```diff
@@ -20,29 +20,22 @@ describe('deprecations', () => {
       host: 'deprecations-test.com'
     })
     return github.integrations.getInstallations({})
-
     .then(() => {
-      console.warn.callCount.should.equal(2)
-
-      simple.restore()
+      expect(console.warn).to.be.calledTwice
```

Comment:
do you know of an alternative way to test for this that is not using a getter? This is currently failing the standard linting

Reply:
ah nevermind, this works:

```js
expect(console.warn.callCount).to.equal(2)
```